src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -847,41 +848,25 @@
             (not keep-tx-data?) (update-in [:wallet :ui :send] dissoc :tx-type))
       :fx [[:dispatch [:navigate-back]]]})))
 
-(rf/reg-event-fx
- :wallet/init-tx-settings
- (fn [{db :db}]
-   {:db (-> db
-            (assoc-in [:wallet :ui :send :custom-tx-settings]
-                      {:max-base-fee   {:low     5
-                                        :current 8.2
-                                        :high    9}
-                       :priority-fee   {:low     0.6
-                                        :high    5.1
-                                        :current 1.1}
-                       :max-gas-amount {:low     30000
-                                        :current 31000}
-                       :nonce          {:last-transaction 21
-                                        :current          22}}))}))
-
 (rf/reg-event-fx
  :wallet/set-max-base-fee
  (fn [{db :db} [value]]
-   {:db (assoc-in db [:wallet :ui :send :custom-tx-settings :max-base-fee :current] value)}))
+   {:db (assoc-in db [:wallet :ui :send :user-tx-settings :max-base-fee] value)}))
```

Comment:
Hey @vkjr, amazing work! 🔥 I have a question about `:user-tx-settings`. It seems we’ll have the same logic for `send`, `swap`, and possibly `bridge` and `wallet connect`. Should they have separate storage locations, or can we use a shared place in app-db to store them? What do you think? Currently, these are stored in `:wallet :ui :send`.

Reply:
Hey @mohsen-ghafouri, thanks!
I think locations should be separate, but logic can be reused, for example by passing the appropriate db prefix path to internal functions.
I mean that we can have different versions of events with different names for swap and send and bridge, but internally they can reuse some functionality that recives `[:wallet :ui :send]` or `[:wallet :ui :path]` as an option.

I've started some refactorings toward this - #22288 Not sure if they will be accepted but we can think of different implementaitons of this idea